container.go

```diff
@@ -410,8 +411,16 @@ func (c *ContainerRequest) ShouldKeepBuiltImage() bool {
 	return c.FromDockerfile.KeepImage
 }
 
-func (c *ContainerRequest) ShouldPrintBuildLog() bool {
-	return c.FromDockerfile.PrintBuildLog
+func (c *ContainerRequest) BuildLogWriter() io.Writer {
+	if c.FromDockerfile.BuildLogWriter == nil {
+		c.FromDockerfile.BuildLogWriter = os.Stderr
+	}
+
+	if c.FromDockerfile.PrintBuildLog {
+		return c.FromDockerfile.BuildLogWriter
+	} else {
+		return io.Discard
+	}
```

Comment:
suggestion: else isn't needed due to return.
```suggestion
	}

	return io.Discard
```

Reply:
Yes, it can be fixed